frontend/packages/dev-console/src/components/topology/operators/operators-data-transformer.ts

```diff
@@ -112,9 +113,14 @@ const OBSModelProps = {
 const isOperatorBackedService = (
   obj: K8sResourceKind,
   installedOperators: ClusterServiceVersionKind[],
+  resources?: TopologyDataResources,
 ): boolean => {
   const kind = _.get(obj, 'metadata.ownerReferences[0].kind', null);
   const ownerUid = _.get(obj, 'metadata.ownerReferences[0].uid');
+  // added this as needs to hide oprator backed if belong to source
+  if (isOperatorBackedKnResource(obj, resources)) {
+    return false;
+  }
```

Comment:
Did you verify that this is necessary? I thought the depicters logic would prevent these items from being added to the model since knative is claiming them as depicted.

Reply:
yes @jeff-phillips-18 for other sources i.e `ping/sinkBinding/container etc` deployments (backing) created are owned by respective eventSources i.e in `ownerReference` of those can see eventSources and these deployments doesn't qualify for Operator backed service as their owner is nor CSV nor they are owned by CRDs owned by CSV. So as you said depicters logic would prevent these items from being added to the model since knative is claiming them as depicted.

but for camel sources(Connectors) deployments (backing) created are not owned by `camelSource` i.e in `ownerReference` of those can see `Integrations` and these deployments qualify for **Operator backed service** as they are owned by CRDs (Integrations) owned by CSV. So depicters logic wouldn't prevent these items from being added to the model as these deployments will be added as children to OperatorBackedService (group).

Let me know if there is a better way to handle it, will look into that. 

cc @christianvogt